Add an explanation for auto saving based on specific languages/file types (#7826)
microsoft/vscode-docs: docs/editor/codebasics.md

  * `onWindowChange` - to save files when the focus moves out of the VS Code window.
* `setting(files.autoSaveDelay)`: Configures the delay in milliseconds when `setting(files.autoSave)` is configured to `afterDelay`. The default is 1000 ms.

> If you want to customize `Auto Save` features for specific languages or file types, you can do so from your `setting.json` by adding language specific rules. For example, you could disable `Auto Save` for LaTeX files like this:

Reviewer: ```suggestion
If you want to customize the `Auto Save` functionality for specific languages or file types, you can do so from the `settings.json` file by adding language-specific rules. 

For example, to disable `Auto Save` for LaTeX files:

```
